src/views/tutorial-library/components/mobile-filters-button/mobile-filters-button.module.css

```diff
@@ -0,0 +1,5 @@
+.root {
+	@media (--dev-dot-hide-mobile-menu) {
+		display: none;
+	}
+}
```

Comment:
Instead of having this tiny file, can we instead pass `"g-show-with-mobile-menu"` as a className to the `Button` in `MobileFiltersButton`? Similar to what we've got going on in `src/pages/tutorials/library.tsx`?